stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.11.3
+version: 1.11.4
```

Comment:
This might need to change to 1.11.6 because of https://github.com/kubernetes/charts/pull/2037 and https://github.com/kubernetes/charts/pull/2080